test/eventHandlers/WorkerInitHandler.test.ts

```diff
@@ -218,14 +219,18 @@ describe('WorkerInitHandler', () => {
             },
         });
 
+        const jsonError = semver.gte(process.versions.node, '19.0.0')

```

Comment:
I support the idea of this PR to preemptively support node 20 as much as possible. But I worry, are we adding additional unnecessary burden here by using node 19 as a proxy for node 20? Are we forcing ourselves to support features/updates in node 19 that may not end up making it to node 20? This change is an example. Are we sure that node 20 will still have this different behavior?

Reply:
Well...it took me all of 10 minutes to fix this unit test. Can you expand on what you mean by burden?

For the record, this PR doesn't mean we actually support anything. Users have to wait for official public preview/GA announcements.